Makefile

```diff
@@ -182,11 +183,13 @@ e2e-tests: $(GINKGO) e2e-substitutions cluster-templates ## This target should b
 
 	$(GINKGO) --timeout=$(GINKGO_TIMEOUT) -v --trace --tags=e2e  \
 		--show-node-events --no-color=$(GINKGO_NOCOLOR) \
+		--fail-fast="$(KEEP_TEST_ENV)" \
 		--junit-report="junit.e2e_suite.1.xml" \
 		--focus="$(GINKGO_FOCUS)" $(_SKIP_ARGS) "$(ROOT_DIR)/$(TEST_DIR)/e2e/" -- \
 		-e2e.artifacts-folder="$(ARTIFACTS)" \
 		-e2e.config="$(E2E_CONF_FILE_ENVSUBST)" \
 		-e2e.skip-resource-cleanup=$(SKIP_CLEANUP) \
+		-e2e.keep-test-environment=$(KEEP_TEST_ENV) \
```

Comment:
What is the difference between `SKIP_CLEANUP` and `KEEP_TEST_ENV`? I see that this PR changes many of the `skipCleanup` to `keepTestEnv` so I'm wondering if we now should remove one or in what way they differ.